### PR TITLE
convert plain css to tailwindcss, refactoring

### PR DESCRIPTION
**Pull request by dermatz**
- refactor existing css
- remove duplicate code
- convert plain css into TailwindCSS
- rearrange css
- unified CSS selectors 
- improved main-menu hover-color for better accessibility
- unified main menu animations
- update dark-modes for gradient-box and tables
- unified line-heights and font-sizes

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,122 +1,120 @@
-import defaultTheme from 'tailwindcss/defaultTheme';
```
Comment (DavidLambauer) [inline tailwind.config.js:1]: Why is that necessary? Just auto format?

**Review by DavidLambauer — APPROVED** [review]
The autoformatting makes it a bit hard to review. But it is most likely fine. Please try to make more atomic changes in the future. 500 lines +- changed are 😱😜😂

**Comment by DavidLambauer** [comment]
Thank you @dermatz !

File: tailwind.config.js (hunk 1 above)
Comment (dermatz) [reply tailwind.config.js:1]: yes typo. Based on .editorconfig. Default is double-quotes.
